docs/install/manual.md

```diff
@@ -67,7 +67,23 @@ chown -R www-data:www-data storage
 chown -R www-data:www-data bootstrap/cache
 ```
 
-## Step 4 : Copy Environment File
+## Step 4: Add cron file
+
+Create a new cron file as `/etc/cron.d/invoiceshelf` with the following line:
+
+```cron
+* * * * * /usr/bin/php /path/to/your/install/artisan schedule:run >> /dev/null 2>&1
+```
```

Comment:
This will only work in Debian derivatives, correct? In other systems `/etc/cron.d/` contains shell scripts. Why not just use `crontab -e`?